examples/ConductorSharp.NoApi/Program.cs

```diff
@@ -49,8 +41,8 @@
                     pipelines.AddValidation();
                 });
 
-            builder.RegisterMediatR(typeof(Program).Assembly);
-            builder.RegisterModule<ConductorModule>();
+            services.AddMediatR(typeof(Program).Assembly);
```

Comment:
This should not be needed anymore, mediatr is registered in the library itself

Reply:
This means also that mediatr extension package can be removed as well

Reply:
Ah, makes sense. Removed it